lib/test/t_Arcsine_std.cxx

```diff
@@ -135,14 +135,6 @@ int main(int argc, char *argv[])
     fullprint << "parameters=" << parameters << std::endl;
     for (UnsignedInteger i = 0; i < 6; ++i) fullprint << "standard moment n=" << i << ", value=" << distribution.getStandardMoment(i) << std::endl;
     fullprint << "Standard representative=" << distribution.getStandardRepresentative()->__str__() << std::endl;
-    // Specific to this distribution
-    NumericalScalar mu = distribution.getMu();
-    fullprint << "mu=" << mu << std::endl;
-    NumericalScalar sigma = distribution.getSigma();
-    fullprint << "sigma=" << sigma << std::endl;
-    Arcsine newDistribution(mu, sigma, Arcsine::MUSIGMA);
-    fullprint << "a from (mu, sigma)=" << newDistribution.getA() << std::endl;
-    fullprint << "b from (mu, sigma)=" << newDistribution.getB() << std::endl;
```

Comment:
Shouldn't we keep tests unchanged to make sure that the compatibility layer is working as expected, and remove them only when methods are dropped?